app/routines/admin/search_security_log.rb

```diff
@@ -132,7 +132,7 @@ def exec(params = {}, options = {})
             type_strings = to_string_array(types)
             sanitized_event_types = Admin::SearchSecurityLog.sanitize_event_types(type_strings)
 
-            @items = @items.where(event_type: sanitized_event_types)
+            @items = @items.where(event_type: sanitized_event_types.flatten)
```

Comment:
Yep, Rails 4 change. It gave a deprecation warning, _I think_ that's the only reason.
```DEPRECATION WARNING: Passing a nested array to Active Record finder methods is deprecated and will be removed. Flatten your array before using it for 'IN' conditions. (called from lib/composite_primary_keys/relation/predicate_builder.rb:32)```